packages/protobuf/src/private/util.ts

```diff
@@ -66,6 +66,12 @@ export interface Util {
     b: T | PlainMessage<T> | undefined | null
   ): boolean;
 
+  /**
+   * Checks if the given value conformst the given type.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  conforms<T extends Message<T>>(type: MessageType<T>, value: any): value is T;
```

Comment:
I'd use `unknown` here instead of `any`. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type

Reply:
Good point! Updated accordingly!